specification/columns/contractappliedcost.md

```diff
@@ -0,0 +1,41 @@
+# Contract Applied Cost
```

Comment:
Is this really applied to the contract? Isn't it applied to a commitment, which is separate from the contract? For Microsoft it is, at least. For instance, MACC is separate from the contract itself. MACC is not tied to a discount. (Although many think it is.) MACC is a negotiating tool sales teams use to establish a commitment, which makes them more comfortable with higher discounts. But the MACC is not the same as the contract.